src/blockchain_state_channel_v1.proto

```diff
@@ -39,6 +39,18 @@ message blockchain_state_channel_packet_v1 {
     packet packet = 1;
     bytes hotspot = 2;
     bytes signature = 3;
+    enum Region {
```

Comment:
I don't fully understand why `Region` enum is defined within the message stanza?

Reply:
Why not?

Reply:
Just curious is all, if it's fine here then let's get this in.

Reply:
merge it